packages/nns/src/canisters/governance/request.converters.ts

```diff
@@ -247,6 +247,9 @@ const fromAction = (action: Action): RawAction => {
                   sns_token_e8s: params.snsTokenE8s,
                   max_participant_icp_e8s: params.maxParticipantIcpE8s,
                   min_icp_e8s: params.minIcpE8s,
+                  neuron_basket_construction_parameters: toNullable(
```

Comment:
should it be added to `response.converters.ts` too?

Reply:
yes, since it's optional it wasn't complaining, but yes.

We should stop converting types back and forth and follow the pattern in sns-js.